docs/user/known-issues.md

```diff
@@ -0,0 +1,158 @@
+# Known Issues
+
+Problems running `Kind`?
+
+This guide is aimed at warning users against known problems and documenting
+their solutions/workarounds, if known.
+
+## Contents
+* [Failing to apply overlay network](#failing-to-apply-overlay-network)
+* [Failure to build node image](#failure-to-build-node-image)
+* [Failure for cluster to properly start](#failure-for-cluster-to-properly-start)
+
+
+## Failing to apply overlay network
+There are two known causes for problems while applying the overlay network
+while building a kind cluster:
+
+* Host machine is behind a proxy
+* Usage of Docker version 18.09
+* Building kind in a Google Compute Engine VM
```

Comment:
I think this was specifically "Google Cloud Console" which is a bit different, I haven't done this recently but the old kind demo gif was actually produced in a GCP VM.

Reply:
A bit of unfamiliarity with Google from my part. Thank you for catching it!